### ISSUE TITLE
FileNotFoundError: [Errno 2] No existe el fichero o el directorio: '/dev/dri/card0'

### ISSUE THREAD

**Issue by UADER-FCyT-SDR** [Idein/py-videocore6#59]
Hi,
Just installed py-videocore6 on a RBPi 4 under bookworm and when running python sgemm.py it says /dev/dri/card is missing, and indeed it is as it no show when doing ls /dev/dri/card*

¿Any ideas or hints on that?

Best Regards, Pedro.

**Comment by notogawa** [comment]
Do you meet [the requirements](https://github.com/Idein/py-videocore6?tab=readme-ov-file#requirements)?
> To access the device, you need to belong to video group or be root user.

**Comment by UADER-FCyT-SDR** [comment]
Yes, indeed. I ran sudo usermod --append --groups video pi during installation and also tried with sudo python......
